test/parallel/test-event-emitter-once.js

```diff
@@ -79,3 +81,14 @@ common.expectsError(() => {
     EventEmitter.prototype.emit.apply(ee, args);
   }
 }
+
+{
+  // verify the promise returning version of `once`
```

Comment:
``   // Verify the promise returning version of `once`. `` (uppecase and period)?